verl/utils/device.py

```diff
@@ -84,3 +84,12 @@ def get_nccl_backend() -> str:
         return "hccl"
     else:
         raise RuntimeError(f"No available nccl backend found on device type {get_device_name()}.")
+
+
+def expandable_segments(enable: bool) -> None:
```

Comment:
Could you rename this function to `set_expandable_segments` to be consistent with Pytorch's naming? Thanks. We can skip CI after renaming